tools/fuels-abi-cli/src/main.rs

```diff
@@ -1,187 +1,185 @@
+//         let format = format!("{}\n", token);
+//         result.push_str(&format);
+//     }
+
+//     Ok(result)
+// }
+
+// fn decode_call_output(path: &str, function_name: &str, data: &str) -> anyhow::Result<String> {
+//     let contract = fs::read_to_string(path)?;
 
-fn code_gen(code: Codegen) -> anyhow::Result<String> {
-    let Codegen {
-        name,
-        input,
-        output,
-        no_std,
-    } = code;
-
-    let contract = fs::read_to_string(input)?;
-    let mut abi = Abigen::new(&name, contract)?;
-
-    if no_std {
-        abi = abi.no_std();
-    }
-
-    let c = abi.generate()?;
-
-    let outfile = output.unwrap_or_else(|| "./abi_code.rs".into());
-    let mut f = fs::File::create(outfile)?;
-    c.write(&mut f)?;
-
-    Ok("File generated".into())
-}
-
-fn encode_params(params: &[String]) -> anyhow::Result<String> {
-    let abi_coder = ABIParser::new();
-
-    Ok(abi_coder.encode_params(params)?)
-}
+//     let abi_coder = ABIParser::new();
+
+//     let decoded = abi_coder.decode(&contract, function_name, data.as_bytes())?;
 
-fn encode_input(path: &str, function_name: &str, values: &[String]) -> anyhow::Result<String> {
-    if values.is_empty() {
-        anyhow::bail!("Values to be encoded shouldn't be empty")
-    }
+//     let mut result: String = String::new();
+//     for res in decoded {
+//         let format = format!("{}\n", res);
+//         result.push_str(&format);
+//     }
 
-    let contract = fs::read_to_string(path)?;
-
-    let mut abi_coder = ABIParser::new();
-
-    let result = abi_coder.encode_with_function_selector(&contract, function_name, values)?;
-
-    Ok(result)
-}
-
-fn decode_params(types: &[String], data: &str) -> anyhow::Result<String> {
-    let abi_coder = ABIParser::new();
-
-    let types: Result<Vec<ParamType>, Error> = types
-        .iter()
-        .map(|s| {
-            ParamType::try_from(&Property {
-                name: "".into(),
-                type_field: s.to_owned(),
-                components: None,
-            })
-        })
-        .collect();
-
-    let data: Vec<u8> = hex::decode(&data)?;
-
-    let decoded = abi_coder.decode_params(&types.unwrap(), &data)?;
-
-    let mut result: String = String::new();
-    for token in decoded {
-        let format = format!("{}\n", token);
-        result.push_str(&format);
-    }
-
-    Ok(result)
-}
-
-fn decode_call_output(path: &str, function_name: &str, data: &str) -> anyhow::Result<String> {
-    let contract = fs::read_to_string(path)?;
-
-    let abi_coder = ABIParser::new();
-
-    let decoded = abi_coder.decode(&contract, function_name, data.as_bytes())?;
-
-    let mut result: String = String::new();
-    for res in decoded {
-        let format = format!("{}\n", res);
-        result.push_str(&format);
-    }
-
-    Ok(result)
-}
+//     Ok(result)
+// }
 
 fn main() -> anyhow::Result<()> {
-    println!("{}", execute(std::env::args())?);
+    // println!("{}", execute(std::env::args())?);
```

Comment:
We might be better off erroring out. But I'm not sure of the audience of these CLI tools. Might be confusing to just have it run and do nothing :)